main.js

```diff
@@ -0,0 +1,109 @@
+import {
+    app,
+    BrowserWindow,
+    // Menu
+} from 'electron';  // eslint-disable-line
+// const path = require('path');
+// const url = require('url');
+// const electron = require('electron')
+
+let win = null;
+
+
+function createWindow() {
+    // Initialize the window to our specified dimensions
+    win = new BrowserWindow({
+        width: 1000,
+        height: 600,
+    });
+    // Specify entry point
+    win.loadURL('http://localhost:3000');
+
+
+    // build menu from template
+    /*  const mainMenu = Menu.buildFromTemplate(mainMenuTemplate);
+      Menu.setApplicationMenu(mainMenu); */
+
+    // Show dev tools
+    win.webContents.openDevTools();
+    // Remove window once app is closed
+    win.on('closed', () => {
+        win = null;
+    });
+}
+
+app.on('ready', () => {
+    createWindow();
+});
+
+app.on('window-all-closed', () => {
+    if (process.platform !== 'darwin') {
+        app.quit();
+    }
```

Comment:
I'm confused, if the platform === darwin, never quit?

Reply:
on a mac it is common for application to stay active until the user input command q or at least that is what I understand from what I read